documentation/Changelog-for-next-generation.md

```diff
@@ -0,0 +1,13 @@
+The next-generation Azure JavaScript libraries introduce a few important changes:
+
+1. Authentication: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are no longer supported. Use package [@azure/identity](https://www.npmjs.com/package/@azure/identity) instead. Select a credential from Azure Identity examples based on the authentication method of your choice. You may find out the new authentication examples [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#authentication)  
+1. Callbacks: Method overloads that use callbacks have been removed and please use Promise instead. You may find out the examples [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#callbacks)  
+1. You could iterate the result of List operations by using the `PagedAsyncIterableIterator` interface, compared with in previous model, you have to make a new request using the link to the next page. You may find out the new list examples [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#list-operations) 
+1. Interface and API change for Long running operations: To check the final result of the Poller object returned by long running operations like `beginCreateOrUpdate`, please use `pollUntilDone` instead of `pollUntilFinished`. To get the final result directly, use the method with the suffix `AndWait` e.g.`beginCreateOrUpdateAndWait`. You may find out the LRO examples [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md#long-running-operations)  
+1. We have merged class XXXClientContext and class XXXClient into one class Client and removed the XXXClientContext.  
```

Comment:
How about just describe  the class of 'XXXClientContext' was removed and all members of it are merged into 'XXXClient'?

Reply:
And is it better to use a real name instead of XXXClient ?

Reply:
I think this is a general changelog link, it's not suppose to be in RP sdk folder. we may not be able to find a real name. maybe we could say for example ?

Reply:
can just use a fake name such as ExampleClientContext